sky/serve/infra_providers.py

```diff
@@ -332,18 +331,19 @@ def _fetch_job_status(self) -> None:
         for cluster_name, info in self.replica_info.items():
             if not info.status_property.should_track_status():
                 continue
+            backend = backends.CloudVmRayBackend()
```

Comment:
Could we add a comment above on why we use backend?

Reply:
Added. Thanks!